Tests/Tests.UWP/TestHelpersUWP.cs

```diff
@@ -0,0 +1,65 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Data.Xml.Xsl;
+using Windows.Data.Xml.Dom;
+using Windows.Storage;
+
+namespace Realms.Tests.UWP
+{
+    static class TestHelpersUWP
+    {
+        public static async Task TransformTestResults(string resultsPath)
+        {
+            TestHelpers.CopyBundledFileToDocuments("nunit3-junit.xslt", "nunit3-junit.xslt");
+            var xsltPath = RealmConfigurationBase.GetPathToRealm("nunit3-junit.xslt");
+            var xsltSf = await StorageFolder.GetFolderFromPathAsync(Path.GetDirectoryName(xsltPath));
+            var localFolderSf = ApplicationData.Current.LocalFolder;
+            var xsltFileSf = await xsltSf.GetFileAsync("nunit3-junit.xslt");
+            var xsltFile = await XmlDocument.LoadFromFileAsync(xsltFileSf);
```

Comment:
These file calls seem overly complicated - I think you can simply call `StorageFolder.GetFileFromPathAsync` without going through the intermediate steps. It's possible that the UWP API have changed, but this is what I was previously using and it seems a bit more readable:

https://github.com/nirinchev/nunit.xamarin/blob/534d8f31c58dee22aa9e7fa2f3c810931446ccbe/src/runner/nunit.runner/Services/XmlFileTransformer.cs#L53-L61